examples/01-Detailed-Examples/05-mesh-exploration.py

```diff
@@ -0,0 +1,209 @@
+# --------------------------------------
+
+# Node IDs
+n_ids = mesh.node_ids
+
+# Element IDs
+e_ids = mesh.element_ids
+
+# Available named selection names
+named_selections = mesh.available_named_selections
+
+# Number of nodes
+n_nodes = mesh.num_nodes
+
+# Number of elements
+n_elements = mesh.num_elements
+
+# Number of named selections
+n_ns = len(named_selections)
+
+# Unit (get and set)
+mesh_unit = mesh.unit
+mesh.unit = "mm"
+
+print(n_ids)
+print(e_ids)
+print(named_selections)
+print(n_nodes)
+print(n_elements)
+
+###############################################################################
+# Get Named Selections
+# --------------------
+ns_list     = mesh.available_named_selections
+first_key = ns_list[0]
+named_selection = mesh.named_selections[first_key]
+
+for k in mesh.named_selections.keys():
+    print(k)
+for v in mesh.named_selections.values():
+    print(v)
+for k, v in mesh.named_selections.items():
+    print(f"{k} = {v}")
+
+###############################################################################
+# Get elements
+# ------------
+
+# Get an element by ID
+el_by_id = mesh.elements.by_id[1]
+
+# Get an element by index
+index = el_by_id.index
+print(mesh.elements[index])
+
+###############################################################################
+# Element Types and Materials
+# ---------------------------
+
+# Get the element types
+el_types = mesh.element_types
+print(el_types)
+
+# Get the materials
+e_materials = mesh.materials
+print(e_materials)
+
+###############################################################################
+# Query information about one particular element
+# ----------------------------------------------
+
+# Get the nodes of an element
+mesh.elements[0].nodes
+
+# Get the node IDs of an element
+mesh.elements[0].node_ids
+
+# Get the nodes of an element
+mesh.elements[0].n_nodes
+
+# Get the type of the element
+mesh.elements[0].type_info
+mesh.elements[0].type_id
+
+# Get the shape of the element
+mesh.elements[0].shape
+
+###############################################################################
+# Get the elemental connectivity
+# ------------------------------
+
+# get node indices from element index
+conn1 = mesh.conn_elem_to_node
+
+# get node IDs from element index
+conn2 = mesh.conn_elem_to_node_id
+
+el_idx_5 = mesh.elements[5]
+# get node IDS from element ID
+conn2.by_id[el_idx_5.id]
+
+###############################################################################
+# Get nodes
+# ---------
+
+# Get a node by ID
+node_by_id = mesh.nodes.by_id[1]
+
+# Get a node by index
+node_by_index = mesh.nodes[0]
+
+# Get the coordinates of all nodes
+print(mesh.coordinates)
+
+###############################################################################
+# Query information about one particular node
+# -------------------------------------------
+
+# Coordinates
+mesh.nodes[1].coordinates
+
+# Get Nodal connectivity
+conn3 = mesh.conn_node_to_elem
```

Comment:
after discussion, we can rename to element_to_node_connectivity and element_to_nodes_ids_connectivity